lib/core/utils/pollyfills.js

```diff
@@ -190,79 +190,53 @@ if (!Array.prototype.some) {
   });
 }
 
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
 if (!Array.from) {
-  Object.defineProperty(Array, 'from', {
-    value: (function() {
-      var toStr = Object.prototype.toString;
-      var isCallable = function(fn) {
-        return (
-          typeof fn === 'function' || toStr.call(fn) === '[object Function]'
-        );
-      };
-      var toInteger = function(value) {
-        var number = Number(value);
-        if (isNaN(number)) {
-          return 0;
-        }
-        if (number === 0 || !isFinite(number)) {
-          return number;
-        }
-        return (number > 0 ? 1 : -1) * Math.floor(Math.abs(number));
-      };
-      var maxSafeInteger = Math.pow(2, 53) - 1;
-      var toLength = function(value) {
-        var len = toInteger(value);
-        return Math.min(Math.max(len, 0), maxSafeInteger);
-      };
-
-      // The length property of the from method is 1.
-      return function from(arrayLike /*, mapFn, thisArg */) {
-        // 1. Let C be the this value.
-        var C = this;
+  Array.from = (function() {
```

Comment:
Our `Array.from` polyfill was not working correctly with `Set` (even though `Set` is supported in IE11...). It always returned an empty array. Updated with the [latest from MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Polyfill)

Reply:
Turns out the MDN polyfill does not even work with native IE11 `Set`...